ui/src/hooks/useTerminal.ts

```diff
@@ -154,6 +154,7 @@ export const useTerminal = async (el: HTMLElement, option: ICallbackOptions): Pr
       case 'TERMINAL_ERROR':
       case 'ERROR': {
         terminal?.write(msg.err);
+        sendEventToLuna('TERMINAL_ERROR', '', lunaId.value, origin.value);
         break;
       }
       case 'MESSAGE_NOTIFY': {
```

Comment:
There seems to be no major irregularity or potential issue with this code snippet. As far as optimization is concerned, there doesn't appear to be any suggested optimization that applies here without additional context about specific performance concerns.

In terms of feedback from you on whether it could "speak" in English:

```html
<template>
  <pre>{{code}}</pre>
</template>

<script>
export default {
  setup() {
    return {code}
  },
}
```
</script>